### ISSUE TITLE
New version of Ehcace 2.0.0 to 2.1.0

### ISSUE THREAD

**Issue by GoogleCodeExporter** [edalquist/ehcache-spring-annotations#20]
```
While using ehcache-spring-annotations, Ehcache logs this message:

1000 [net.sf.ehcache.CacheManager@3e97df] INFO
net.sf.ehcache.util.UpdateChecker - New update(s) found: 2.1.0
[http://ehcache.org/news.html]. Please check http://ehcache.org for the
latest version.

Would it be possible to migrate to the newest version of Ehcache?
```

Original issue reported on code.google.com by `erikvani...@gmail.com` on 20 May 2010 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
You can do this for your project by adding an ehcache-core dependency for 
version
2.1.0 which will override the version this library includes by default.

We will take a look at compiling against 2.1.0 for our 1.1 release
```

Original comment by `eric.dalquist` on 20 May 2010 at 12:24
- Added labels: **Type-Enhancement**, **Milestone-Release1.1.0**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric.dalquist` on 29 May 2010 at 1:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
We had to back this change out. There is a bug in Ehcache 2.1.0 which makes our 
tests fail: https://jira.terracotta.org/jira/browse/EHC-734

We will reconsider when Ehcache fixes that issue.
```

Original comment by `eric.dalquist` on 11 Jun 2010 at 6:39
- Changed state: **WontFix**
- Removed labels: **Milestone-Release1.1.0**

**Comment by GoogleCodeExporter** [comment]
```
Hi Erik,

This bug has fixed in Ehcache 2.2.0, so would it be possible to migrate to 
Ehcahe 2.2.0?

```

Original comment by `www.icb....@gmail.com` on 8 Nov 2010 at 2:27

**Comment by GoogleCodeExporter** [comment]
```
The library works with all versions of ehcache from 1.6.0 to 2.2.0. You simply 
need to import the newer ehcache dependency in your project.
```

Original comment by `eric.dalquist` on 8 Nov 2010 at 3:18
